lib/util.js

```diff
@@ -0,0 +1,39 @@
+const { Op } = require("sequelize");
+const db = require("../sequelize/db");
+const pg = require("../database/postgres/pg");
+const es = require("../database/elasticsearch/elastic");
+const logger = require("./log")(__filename);
+
+const util = {};
+
+util.cleanAnonymous = async () => {
+  logger.info("Checking expired anonymous accounts...");
+  try {
+    const { Accounts } = db.getModels();
+    const userAccount = await Accounts.findAll({
+      attributes: ["id", "username"],
+      where: {
+        [Op.and]: [
+          {
+            createdAt: {
+              [Op.lt]: new Date(new Date() - 5 * 24 * 60 * 60 * 1000),
+            },
+          },
+          { email: null },
+        ],
+      },
+    });
+    logger.info(`${userAccount.length} expired accounts are found`);
+    userAccount.forEach(async (e) => {
```

Comment:
Using async await in a `forEach` loop won't work. This will cause multiple asynchronous operations to run, but the problem is that those operations won't be guaranteed to finish before line 34(`logger.info`) gets read.

This stackOverflow answer has really good explanation and solutions for this case.
https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop?rq=1

Reply:
That's a really good feedback. I'm fixing it soon